MultiServer.py

```diff
@@ -1102,7 +1110,15 @@ def format_hint(ctx: Context, team: int, hint: NetUtils.Hint) -> str:
 
     if hint.entrance:
         text += f" at {hint.entrance}"
-    return text + (". (found)" if hint.found else ("." if hint.priority else ". (non-priority)"))
+    
+    status_names: typing.Dict[NetUtils.HintStatus, str] = {
```

Comment:
Should this be pulled out of the method so it doesn't get intiialized every time?

Reply:
it can probably be cleaned up and/or better organized, and probably even shared to reduce duplication of a couple of them, but I have no idea how people would want that organized, where it should go, etc.

Reply:
Even without any organisation, just placing it right above the method would be a performance increase, although I have no idea by how much. Again a case of "If we had a compiler we'd never need to even think about that" lmao